pages/common/sdiff.md

```diff
@@ -0,0 +1,23 @@
+# sdiff
+
+> Compare the differences and/or merge between two files.
+
+- Compare 2 files:
+
+`sdiff {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files, ignoring all tabs and whitespace:
+
+`sdiff -W {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files, ignoring whitespace at the end of lines:
+
+`sdiff -Z {{path/to/first_file}} {{path/to/second_file}}`
+
+- Case insensitively compare 2 files:
+
+`sdiff -i {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files and then merge, creating a new file:
```

Comment:
Compare and then merge the output to a new file

Reply:
I also just change this by the suggestion from @sbrl . I personally like the one that he suggest more.

Reply:
@sbrl suggested to avoid the repeating "and", "then". My suggestion is slightly different.

Reply:
Quite not sure, for me both is ok. Let ask for third opinion.

@sbrl what do you think?

Reply:
Ah, I see. I'm not quite sure about the `merge the output to` bit specifically there. What about this?
```suggestion
- Compare and then merge, writing the output to a new file:
```

Reply:
I like this one, do you agree with this? @agnivade

Reply:
I see your point. It doesn't really merge the output, rather it merges the output of the comparisons of the 2 files. However, I really wanted to avoid the comma in the sentence, which kinda stymies the flow. But I don't see a better way. Let's go with your one.